src/components/connections/Connection.ts

```diff
@@ -98,22 +98,15 @@ export class InfluxDBTreeDataProvider
     if (connections) {
       const activeID = Status.Current?.id;
       for (const id of Object.keys(connections)) {
-        let active = false;
-        if (
-          activeID === id ||
-          (activeID === undefined && connections[id].isDefault)
-        ) {
-          active = true;
-          Status.Current = connections[id];
-        }
+        connections[id].isActive = activeID === id || connections[id].isActive;
```

Comment:
could this be connections[id].isActive = activeID === id so we don't end up with two active connection?

Reply:
two connections won't really happen, since each time we created a new one, all the active connections will be flipped. `|| connections[id].isActive` is useful, when we restart vscode, the activeID will be always undefined,